source/recipes/product-vulnerability-incident.rst

```diff
@@ -0,0 +1,112 @@
+Mattermost Recipe: How to use Mattermost for Product Vulnerability Incidents
+=====================
+
+Problem
+---------------
+
+After getting access to your Mattermost instance, you’d like to set up a workflow for security incident handling using
+Mattermost. Click `here<https://mattermost.com/deploy/>`__ if you want to learn more about how to deploy your own Mattermost instance.
+
+Solution
+---------------------
+
+The following guide will walk through the set up of a product security incident room using the Mattermost `Playbooks<https://docs.mattermost.com/guides/playbooks.html>`__ and
+`Channels<https://docs.mattermost.com/guides/channels.html>`__ with the voice calling and screen sharing functionalities.
+
+1. Workspace Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For Cloud customers, all the functionalities will work out-of-the-box with no technical setup. Simply invite your team
+members to your workspace and move onto the next section For self-hosted installations, refer to `this admin guide<https://docs.mattermost.com/configure/configuration-settings.html#calls-beta>`__ to
```

Comment:
```suggestion
members to your workspace and move onto the next section. 

For self-hosted installations, refer to `this admin guide<https://docs.mattermost.com/configure/configuration-settings.html#calls-beta>`__ to
```

Reply:
Covered by Justine's review I think, can you double check? Thanks!